packages/htmlbars-compiler/tests/html-compiler-test.js

```diff
@@ -1012,6 +1013,8 @@ test("A helpful error message is provided for mismatched start/end tags", functi
   }, /Closing tag `div` \(on line 5\) did not match last open tag `p` \(on line 2\)\./);
 });
 
+if (innerHTMLHandlesNewlines) {
```

Comment:
These skipped tests don't actually have anything to do with `innerHTML`, so this should probably be a separate feature detect (also involving newlines in IE8, but not `innerHTML`), or rename this feature to be more generic.  That is, unless we find a way to get those tests passing so they don't have to be skipped.